requirements/dev.in

```diff
@@ -6,6 +6,7 @@ factory-boy
 flake8
 furo
 isort
+jsonschema
```

Comment:
`jsonschema` is python 3.7+ as of v3.0.2.

---
**But:** If you want to just bite the bullet and start the 3.6 deprecation, I'm here for it.

Reply:
@jstvz I had already coded up a workaround and didn't want to block on 3.6 deprecation.